test/performace_check/test_fftw.py

```diff
@@ -12,71 +12,96 @@
 
 @profile
 def fftw_comp_2D():
+    # create random variables
+    Nseg = 47
+    Npts = 3600 * 20
+
+    a = np.random.rand(Nseg, Npts) * 35
+    b = np.random.rand(Nseg, Npts) * 20
+    c = a + 1j * b
+
     # compare efficiency of three fftw package
     t0 = time.time()
+    fft0 = pyfftw.interfaces.numpy_fft.fft(a)
+    _ = fft0
```

Comment:
I think this should work:
```suggestion
    _ = pyfftw.interfaces.numpy_fft.fft(a)
```
same below ...